turtle_tf2_py/turtle_tf2_py/static_turtle_tf2_broadcaster.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import rclpy
+from rclpy.node import Node
+from geometry_msgs.msg import TransformStamped
+from tf2_ros.static_transform_broadcaster import StaticTransformBroadcaster
+import transforms3d
```

Comment:
This depend can be installed with `rosdep`, [here it's the dependency](https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L5151). You should add this to the package.xml

Reply:
Am I correct that I should just include the following line?

`<exec_depend>python-transforms3d-pip</exec_depend>`

Reply:
Looks good to me. You can always reference the rosdistro repo to check if you're correct: 
https://github.com/ros/rosdistro/blob/6435e39c61db1e27851efca84c1f2dd63ee72269/rosdep/python.yaml#L5151